packages/teleport/src/cluster/components/Audit/Audit.story.test.tsx

```diff
@@ -8,3 +8,10 @@ test('overflow', async () => {
 
   expect(container.firstChild).toMatchSnapshot();
 });
+
+test('null response still renders table', async () => {
```

Comment:
This is a good example when it's bad to use snapshots for testing. If you prefer to unit-test it, please unit-test the service itself as it should never return null.

Reply:
revised